app/src/pages/admin/projectStatusPage/statusPageContent/statusPageContent.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
+import classNames from 'classnames/bind';
+import { fetch } from 'common/utils';
+import { SpinningPreloader } from 'components/preloaders/spinningPreloader';
+import { statusPageWidgets, activityItem } from './statusPageWidgets';
+import { StatusPageItem } from './statusPageItem';
+import styles from './statusPageContent.scss';
+
+const cx = classNames.bind(styles);
+
+@injectIntl
+export class StatusPageContent extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    projectId: PropTypes.string.isRequired,
+    interval: PropTypes.string,
+  };
+
+  static defaultProps = {
+    interval: '',
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
You may define default state as a class field, so you won't need constructor.

Reply:
Fixed